sphinx/conf.py

```diff
@@ -394,5 +394,7 @@ def getValue(value):
     r'https://www.excelitas.com/*', # 403 Client Error: Forbidden
     r'https://www.sas.upenn.edu/*', # certificate issuer cannot be verified
     r'https://www.microsoft.com/en-us/download/details.aspx?id=48145', # 403 Client Error: Forbidden
-    r'https://www.agilent.com/home' # 403 Client Error: Forbidden
+    r'https://www.agilent.com/home', # 403 Client Error: Forbidden
+    r'https://www.mathworks.com', # 403 Client Error: Forbidden
```

Comment:
403 errors still appear in https://merge-ci.openmicroscopy.org/jenkins/job/BIOFORMATS-linkcheck/287/console. Looking at other examples above, it might be that `https://www.mathworks.com/*` needs to be used instead?